### PR TITLE
docs(aio): add short section on built-in validators

### PR DESCRIPTION
**Pull request by kapunahelewong**
**Please check if the PR fulfills these requirements**
- [] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Other... Please describe: A paragraph about the built-in validators.
```

**What is the current behavior?** (You can also link to an open issue here)
There is nothing that mentions the built-in validators explicitly.

**What is the new behavior?**
Mentioning them specifically and linking to the API reference page.

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: aio/content/guide/form-validation.md [hunk 1]
```diff
@@ -426,13 +429,11 @@ The reactive approach does not use data binding to move data into and out of the
 That's all in code.
 
 
-<div class="l-sub-section">
-
-
+<!--<div class="l-sub-section">
```
Comment (wardbell) [inline aio/content/guide/form-validation.md]: Let's remove this rather than comment it out.

File: aio/content/guide/form-validation.md [hunk 2]
```diff
@@ -534,15 +535,15 @@ The `onSubmit()` method simply replaces the `hero` object with the combined valu
 
 
 
-<div class="l-sub-section">
+<!--<div class="l-sub-section">
```
Comment (wardbell) [inline aio/content/guide/form-validation.md]: Delete rather than comment out.

**Review by wardbell — CHANGES_REQUESTED** [review]
Good work. Appropriate for the small window of time that I allowed for this cleanup.

I've left some comments about a few small things plus what we talked about. Happy to approve when you're done.

File: aio/content/guide/form-validation.md [hunk 3]
```diff
@@ -5,23 +5,17 @@
 
 Improve overall data quality by validating user input for accuracy and completeness.
```
Comment (kapunahelewong) [inline aio/content/guide/form-validation.md]: Aha. I'd removed one and missed this one. Done!

File: aio/content/guide/form-validation.md [hunk 4]
```diff
@@ -34,17 +28,29 @@ and the [Reactive Forms](guide/reactive-forms) guides.
...
+that help you check common user input in forms. In addition to the built-in 
+validators covered here of `minlength`, `maxlength`, 
+and `required`, there are others such as `min`, `max`, `email` and `pattern` 
+for template driven as well as reactive forms. 
+For a full list of built-in validators, 
+see the [Validators](api/forms/Validators) API reference.
+
+
+
 {@a template1}
```
Comment (kapunahelewong) [inline aio/content/guide/form-validation.md]: Certainly. Done.

File: aio/content/guide/form-validation.md [hunk 5]
```diff
@@ -402,7 +405,7 @@ validating happens in code.
 * `required` remains, not for validation purposes (that's in the code),
 but rather for css styling and accessibility.
 
-
+<!--
```
Comment (kapunahelewong) [inline aio/content/guide/form-validation.md]: Got it. Done.

File: aio/content/guide/form-validation.md (hunk 1 above)
Comment (kapunahelewong) [inline aio/content/guide/form-validation.md]: Done.

File: aio/content/guide/form-validation.md (hunk 2 above)
Comment (kapunahelewong) [inline aio/content/guide/form-validation.md]: Done.

**Review by wardbell — APPROVED** [review]

**Pull request merged**